docs/configuration.rst

```diff
@@ -104,7 +104,9 @@ below:
      - ``http://localhost:8126``
      - The URL to use to connect the Datadog agent. The url can starts with
        ``http://`` to connect using HTTP or with ``unix://`` to use a Unix
-       Domain Socket.
+       Domain Socket.   
+       Example for http url: ``DD_TRACE_AGENT_URL=localhost:8126``
```

Comment:
don't we need `http://` ?

Reply:
Changed it, and also added that the default UDS is the default for DD_TRACE_AGENT_URL  if available to the configuration.rst file.